tests/golden/legacy/cert-manager/cert-manager/10_cert_manager/cert-manager/templates/crds.yaml

```diff
@@ -2,14 +2,12 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    helm.sh/resource-policy: keep
+    argocd.argoproj.io/sync-options: Prune=false
```

Comment:
What's the prune for?

Reply:
Just for fun?

Reply:
```
beer.vshn.io/color: dark-amber
```

Reply:
Since the CRDs are "normal" templates in this Chart, I don't wan't them to be accidentally purged.

Reply:
Eh fair enough. Not like we can uninstall this chart on our clusters anyways.